src/components/domain/RiskSeasonalMap/i18n.json

```diff
@@ -9,7 +9,7 @@
         "hazardsTypeLegendLabel": "Type of hazards",
         "peopleExposedOptionLabel": "People Exposed",
         "peopleAtRiskOptionLabel": "People at Risk of Displacement",
-        "riskScoreOptionLabel": "Risk Score",
+        "riskScoreOptionLabel": "INFORM Risk Score",
```

Comment:
We are using both the `INFORM` and `WFP` for this one. So we should keep it as `Risk Score`